test/functional/columns/value-transformer/entity/Post.ts

```diff
@@ -27,4 +59,6 @@ export class Post {
     @Column({ type: String, transformer: new TagTransformer() })
     tags: string[];
 
+    @Column({ type: String, transformer: new ComplexTransformer(), nullable: true })
+    complex: Complex | null;
```

Comment:
Is it possible to add this to a different entity in a different entity file?

That way we're not muddying up the original tests